src/Integration.Vsix/Telemetry/GeneratedCode/SqmOleCommandTarget.cs

```diff
@@ -33,6 +33,7 @@ internal enum SonarLintSqmCommandIds
         ToggleShowAllProjectsCommandCommandId = 0x306,
         DontWarnAgainCommandCommandId = 0x307,
         FixConflictsCommandCommandId = 0x308,
+        UpdateBindingCommandFromErrorListCommandId = 0x400,
```

Comment:
I don't think its a requirement (we don't look at streams of events). I suspected that Jean-Marc just filters based on the commandset guid and then groups (+count) based on the ids. 
In general 100 commands is a lot, more complex projects have significantly less commands that that. 
Specifically to your case - is it the practices to use the same command set guid for fake and real VS commands ? [I'd check with Jean-Marc since he'll know without digging around too much]. If you need a different commands set (which I suspect you should), in that case you case use any range you like :)